guides/UPGRADING.md

```diff
@@ -0,0 +1,46 @@
+Technology Upgrading Policy
+=======================
+
+## Overview
+
+Our approach to version upgrades is contingent on the technology in question. Details of which are outlined below, per technology and language.
+
+If you notice any upgrades that are not covered by this policy (including major versions of DBs becoming available), please
+raise them on the 20% board for triage.
+
+We don’t update dependencies in legacy apps unless there is a high or severe security vulnerability present (babbage, zebedee, train, brian,
+legacy florence).
+
+### Go
+
+- When a new Go version is released, wait for a patch release (i.e., `0.X.1`) before upgrading. For instance, if it's a choice between Go `1.15.11` and `1.16.0`, go with `1.15.11`.
```

Comment:
```suggestion
- When a new Go version is released, wait for a patch release (i.e., `0.X.1`) before upgrading. For instance, if it's a choice between Go `1.15.11` and `1.16.0`, choose `1.15.11`.
```

Just to be consistent with Cats suggestion.

Reply:
Updated